qiita_db/portal.py

```diff
@@ -272,8 +272,20 @@ def remove_studies(self, studies):
             raise ValueError('Can not remove from main QIITA portal!')
         self._check_studies(studies)
 
-        # Clean list of studies down to ones associated with portal already
         conn_handler = SQLConnectionHandler()
+        # Make sure study not used in analysis in portal
+        sql = """SELECT DISTINCT study_id from qiita.study_processed_data
+                 JOIN qiita.analysis_sample USING (processed_data_id)
+                 JOIN qiita.analysis_portal USING (analysis_id)
+                 WHERE portal_type_id = %s AND study_id IN %s"""
+        analysed = [x[0] for x in conn_handler.execute_fetchall(
+            sql, [self.id, tuple(studies)])]
+        if analysed:
+            raise QiitaDBError("The following studies are used in an analysis "
```

Comment:
Can you add a description in the docstring of why this exception could get raised?